pkg/virt-handler/vm_test.go

```diff
@@ -2594,12 +2562,19 @@ var _ = Describe("VirtualMachineInstance", func() {
 					Expect(condition.Status).To(Equal(k8sv1.ConditionTrue))
 				}
 			},
-				Entry("PVC with access mode ReadWriteOne", fsPvcVolume, k8sv1.ReadWriteOnce, true, fmt.Sprintf(errorMsg, fsPvcVolume.PersistentVolumeClaim.PersistentVolumeClaimVolumeSource.ClaimName)),
-				Entry("PVC with access mode ReadOnlyMany", fsPvcVolume, k8sv1.ReadOnlyMany, true, fmt.Sprintf(errorMsg, fsPvcVolume.PersistentVolumeClaim.PersistentVolumeClaimVolumeSource.ClaimName)),
-				Entry("PVC with access mode ReadWriteMany", fsPvcVolume, k8sv1.ReadWriteMany, false, ""),
-				Entry("DV with access mode ReadWriteMany", fsDvVolume, k8sv1.ReadWriteMany, false, ""),
-				Entry("DV with access mode ReadOnlyMany", fsDvVolume, k8sv1.ReadOnlyMany, true, fmt.Sprintf(errorMsg, fsDvVolume.DataVolume.Name)),
-				Entry("DV with access mode ReadWriteOnce", fsDvVolume, k8sv1.ReadWriteOnce, true, fmt.Sprintf(errorMsg, fsDvVolume.DataVolume.Name)),
+
+				Entry("PVC with access mode ReadWriteOne", libvmi.WithFilesystemPVC("VIRTIOFS"),
+					k8sv1.ReadWriteOnce, true, fmt.Sprintf(errorMsg, "VIRTIOFS")),
+				Entry("PVC with access mode ReadOnlyMany", libvmi.WithFilesystemPVC("VIRTIOFS"),
+					k8sv1.ReadOnlyMany, true, fmt.Sprintf(errorMsg, "VIRTIOFS")),
+				Entry("PVC with access mode ReadWriteMany", libvmi.WithFilesystemPVC("VIRTIOFS"),
+					k8sv1.ReadWriteMany, false, ""),
```

Comment:
You could infer `shouldFail` from `errorMsg != ""`

Reply:
Done!